src/ui/templates/icons/voiceSearchIcon.hbs

```diff
@@ -1,17 +1,17 @@
-<div class="yxt-SearchBar-voiceIconWrapper Icon--lg">
-  <div class="yxt-SearchBar-micIconWrapper {{#if customMicIconUrl}}yxt-SearchBar-CustomMicIcon{{/if}}">
+<span class="yxt-SearchBar-voiceIconWrapper Icon--lg">
```

Comment:
I think all the new spans in this file would also need the styling to be added. or we could revert the change for now if the techops didn't need this icon and if we're making a follow-up item to address any remaining similar violations

Reply:
i think the former is the approach @benmcginnis was wanting for this specific PR, I'll make a follow-up item for addressing the remaining violations.